src/summary/cards/summary-card.component.tsx

```diff
@@ -1,31 +1,49 @@
 import React, { ReactChildren } from "react";
 import styles from "./summary-card.css";
-import { match } from "react-router";
+import { match, Link } from "react-router-dom";
+
 import { Trans, useTranslation } from "react-i18next";
 
 export default function SummaryCard(props: SummaryCardProps) {
   const { t } = useTranslation();
   return (
     <div style={props.styles} className={`omrs-card ${styles.card}`}>
-      <div className={styles.header}>
-        <div className={styles.title}>
-          <h2 className={`omrs-margin-0`}>{props.name}</h2>
-          <svg className="omrs-icon" fill="rgba(0, 0, 0, 0.54)">
-            <use xlinkHref="#omrs-icon-chevron-right" />
-          </svg>
+      <div className={`${styles.header}`}>
+        <div className={styles.item_a}>
```

Comment:
could you give this css class a more descriptive name, `item_a` is not very helpful